BSB_lan.ino

```diff
@@ -4083,11 +4095,11 @@ int set(int line      // the ProgNr of the heater parameter
 
   uint8_t type=get_cmdtbl_type(i);
 #if defined(__AVR__)
-  uint8_t enable_byte=pgm_read_float_far(pgm_get_far_address(optbl[0].enable_byte) + type * sizeof(optbl[0]));
-  uint8_t operand=pgm_read_float_far(pgm_get_far_address(optbl[0].operand) + type * sizeof(optbl[0]));
+  uint8_t enable_byte=pgm_read_byte_far(pgm_get_far_address(optbl[0].enable_byte) + type * sizeof(optbl[0]));
+  float operand=pgm_read_float_far(pgm_get_far_address(optbl[0].operand) + type * sizeof(optbl[0]));
```

Comment:
i changing type of operand from uint8_t  to float.

Reply:
Ok, but the change in code is from float to byte (uint8_t) - or did you mean otherwise?

Reply:
Sorry, I didn't read correctly, all is fine!